proxmox/resource_vm_qemu.go

```diff
@@ -1662,7 +1667,7 @@ func resourceVmQemuRead(ctx context.Context, d *schema.ResourceData, meta interf
 
 	// need to set cache because proxmox-api-go requires a value for cache but doesn't return a value for
 	// it when it is empty. thus if cache is "" then we should insert "none" instead for consistency
-	var rxCloudInitDrive = regexp.MustCompile(`^.*-cloudinit.*`)
+	var rxCloudInitDrive = regexp.MustCompile(`^vm-[0-9]+-cloudinit.raw$`)
```

Comment:
I think that should be left as it is, .raw extension is only present with storage that supports it

Reply:
ah! cheers! ran with debug a now I see it.